cli_ftd/task_config.h

```diff
@@ -69,7 +69,7 @@
  * Priority of the Application task.
  */
 #ifndef TASK_CONFIG_CLI_TASK_PRIORITY
-#define TASK_CONFIG_CLI_TASK_PRIORITY 3
+#define TASK_CONFIG_CLI_TASK_PRIORITY 4
```

Comment:
Shift this to your .c udp files
Make it internal static const

Reply:
This is CLI task specific and TI's CLI code.